auth/app/build.gradle

```
@@ -40,7 +40,8 @@ dependencies {
     implementation 'com.google.android.material:material:1.1.0'
 
     // Firebase Authentication
-    implementation 'com.google.firebase:firebase-auth:19.2.0'
+    implementation 'com.google.firebase:firebase-core:17.2.3'
```

Shouldn't we use the analytics library instead?

```suggestion
    implementation 'com.google.firebase:firebase-analytics:17.2.3'
```

---

Good catch!